config/routes.rb

```diff
@@ -1,13 +1,17 @@
 Rails.application.routes.draw do
-
   root 'posts#index'
-
+  
+  get '/add_friend/:friend_id', to: 'friendships#create', as: :send_request
+  put '/accept_friend/:sender_id', to: 'friendships#accept', as: :accept_request
```

Comment:
You should use `post` instead of `put`. Read more [here](https://restfulapi.net/rest-put-vs-post/).